components/swatchgroup/stories/swatchgroup.stories.js

```diff
@@ -68,15 +115,94 @@ export default {
 	},
 };
 
-export const Default = SwatchGroup.bind({});
+/**
+ * The default swatch group has regular density.
+ */
+export const Default = SwatchgroupGroup.bind({});
 Default.args = {};
 
+// ********* DOCS ONLY ********* //
+export const Compact = Template.bind({});
```

Comment:
Same! 
I was also thinking it'd be nice if "Densities" was under a single story, showing default, compact, and spacious next to each other. Not totally necessary right now for the migration but wanted to throw out that suggestion.

Reply:
How about this? https://github.com/adobe/spectrum-css/pull/2925/commits/163192d9d9d77125cf695af3fc93d51143577914

The story names should be changed for both table and swatchgroup now. I didn't immediately add the single story for densities, since accordion wasn't like that, but I certainly can if you both think that's better!